css-brochure-form/styles.css

```diff
@@ -0,0 +1,49 @@
+.lmore{
+  color:rgb(255, 123, 0);
+  padding-left: 110px;
+  font-family: 'Brandon Grotesque';
+  margin-bottom: 0px;
+  font-style: normal;
+  font-weight: 500;
+}
+div{
+  padding-left: 400PX;
+}
+label{
+  display: block;
+  font-size: 8px;
+  padding-top: 10px;
+  padding-left:5px;
+  font-family: 'AtlasGrotesk';
+  font-weight: 500px;
+}
+input{
+  border-radius: 5px;
+  height:30px
+}
+#submit{
+  display:block;
+  background-color: rgb(240, 60, 60);
+  color: white;
+  margin-top: 10px;
+  margin-left: 350px;
```

Comment:
Instead of using margin to push the button to the right, you should put the button in a `div`, and set that `div` to `text-align: right`. That will push the button to the right side without the need for margin.